.github/workflows/build-and-test.yml

```diff
@@ -14,6 +14,12 @@ jobs:
         image: ghcr.io/deploygate/gradle-plugin-mock-server
         ports:
           - 3000/tcp
+        options: >-
```

Comment:
We should add healcheck command. Sometimes the rails app hasn't started before executing unit tests.